provision/minikube/keycloak/templates/keycloak.yaml

```diff
@@ -67,7 +67,7 @@ spec:
     - name: storage-hotrod-password
       value: admin
 {{ end }}
-{{ if .Values.disableCaches }}
+{{- if .Values.disableCaches }}
```

Comment:
so was this never working before ? @mhajas

Reply:
It worked. 

I added `-` because there is no empty line in the resulting yaml. I added it because it didn't look good with the new line, see: https://www.keycloak.org/high-availability/deploy-keycloak-kubernetes there is an empty line below `spec:`

Search for `{{-` here: https://helm.sh/docs/chart_template_guide/control_structures/  for more details

Reply:
thanks @mhajas !